cylc/flow/cfgspec/globalcfg.py

```diff
@@ -495,14 +495,59 @@
                nedit
         ''')
 
-    with Conf('platforms'):
-        with Conf('<platform name>') as Platform:
+    with Conf('platforms', desc='''
+        .. versionadded:: 8.0.0
+
+        Platforms allow you to define compute resources available at your
+        site.
+
+        A platform must allow interaction with the same task job from *any*
+        of its hosts.
+    '''):
+        with Conf('<platform name>', desc='''
+            .. versionadded:: 8.0.0
+
+               Many of the items in platform definitions have been moved from
+               :cylc:conf:`task definitions <flow.cylc[runtime][<namespace>]>`.
+
+            Platform names can be regular expressions: If you have a set of
+            compute resources such as ``bigmachine1, bigmachine2`` or
+            ``desktop0000, .., desktop9999`` one would define platforms with
+            names ``[[bigmachine[12]]]`` and ``[[desktop[0-9]{4}]]``.
+
+            .. note::
+
+               Each possible match to the definition regular expression is
+               considered a separate platform.
+
+               If you had a supercomputer with multiple login nodes this would
+               be a single platform with multiple :cylc:conf:`hosts`
+
+        ''') as Platform:
+            Conf('hosts', VDR.V_STRING_LIST, desc='''
+                .. versionadded:: 8.0.0
+
+                A list of hosts from which the job host can be selected using
+                :cylc:conf:`[..][selection]method`.
+
+                All hosts should share a file system.
+            ''')
             Conf('job runner', VDR.V_STRING, 'background', desc='''
+
+                .. versionchanged:: 8.0.0
+
+                   This item was formerly known as ``batch system``.
+
                 The batch system/job submit method used to run jobs on the
                 platform, e.g., ``background``, ``at``, ``slurm``,
                 ``loadleveler``...
             ''')
-            Conf('job runner command template', VDR.V_STRING)
+            Conf('job runner command template', VDR.V_STRING, desc='''
+                .. versionchanged:: 8.0.0
+
+                   This item was formerly known as
+                   ``batch system command template``.
```

Comment:
The wording is OK, but IMO slightly better to be more direct and explicit as in my previous suggestion for `job runner`.